Tribler/community/tunnel/crypto/tunnelcrypto.py

```diff
@@ -43,13 +43,17 @@ def verify_and_generate_shared_secret(self, dh_secret, dh_received, auth, B):
         return shared_secret
 
     def generate_session_keys(self, shared_secret):
+        # TODO hkdf???
```

Comment:
Btw, this was all implemented after this issue
https://github.com/Tribler/tribler/issues/1066